docs/sop.rst

```diff
@@ -0,0 +1,135 @@
+Operating Clowder
+=================
+
+Two primary aspects of operating Clowder: Operating the apps managed by Clowder and operating
+Clowder itself.
+
+Contrary to many other deployment models where apps configure themselves, an application that is
+managed by Clowder will have configuration presented to it and is expected to use it. Clowder
+governs many different aspects of an application's configuration, from defining the port it should
+listen on, to metrics, kafka topics, service hostnames, and more. In this way a common configuration
+format is presented to the application no matter the envrionment it is running in, enabling a far
+easier development experience.
+
+Operating Apps Managed by Clowder
+---------------------------------
+
+ClowdEnvironment
+++++++++++++++++
+
+**abbreviated to ``env`` in k8s**
+
+The ``ClowdEnvironment`` CRD is responsible for configuring the environment that the Clowder enabled
```

Comment:
what is an environment comprised of?  From Clowder's perspective, an environment is made up of the configuration and deployments needed to operate a complete clouddot environment.  I guess just saying "environment" here is kind of redundant.